### PR TITLE
 docs(en): added linear-search

### PR DESCRIPTION
**Pull request by CaptMadhan**
**What kind of change does this PR introduce?** (check at least one)
- [ ] Bugfix
- [ ] New algorithm
- [ ] Optimization in previous algorithms
- [ ] Code style update
- [ ] Refactor
- [x] Documentation
- [ ] Other, please describe:

Added Linear Search Algorithm documentation in English according to the template given.

### CODE REVIEW

File: docs/en/Searching/Linear-Search.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Linear Search
+
+**linear search**, also known as **Sequential search** is a method for finding an element within a list. A linear search sequentially checks each element of the list until it finds an element that matches the target value. If the algorithm reaches the end of the list, the search terminates unsuccessfully.
+1. Time Complexity  : O(n)
+2. Space Complexity : O(1)
+3. Applications     : practical when the list has only a few elements,
+                      when performing a single search in an un-ordered list.
+4. Founder's Name   : Anatole Beck and Donald J. Newman (1970)
```
Comment (ming-tsai) [inline docs/en/Searching/Linear-Search.md:8]:
```suggestion
1. Time Complexity: O(n)
2. Space Complexity: O(1)
3. Applications: practical when the list has only a few elements when performing a single search in an unordered list.
4. Founder's Name: Anatole Beck and Donald J. Newman (1970)
```

File: docs/en/Searching/Linear-Search.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+## Pseudocode
+
+**procedure linear_search (list, value)**
+
+   for each item in the list
+      if match item == value
+         return the item's location
+      end if
+   end for
```
Comment (ming-tsai) [inline docs/en/Searching/Linear-Search.md:31]: bundle with these code in as code:
```
   for each item in the list
      if match item == value
         return the item's location
      end if
   end for
```

File: docs/en/Searching/Linear-Search.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+end procedure
```
Comment (ming-tsai) [inline docs/en/Searching/Linear-Search.md:33]: Are you missing the `start procedure`?

**Review by ming-tsai — CHANGES_REQUESTED** [review]
Hi @CaptMadhan,
Nice work, but please fix the spelling problem and apply some suggestions
image

**Comment by CaptMadhan** [comment]
Please check now, Resolved the following:
Bundled the procedure as code,
corrected the error in Algorithm Description

**Review by ming-tsai — APPROVED** [review]

**Pull request merged**